File: Expressions/financial_calc.py
# Tatiana Susov, Financial Calculator Python
greeting="Welcome to your personal financial/budget calculator, lets begin!"
print(greeting)
income = float(input("What is your monthly income?\n"))
rent = float(input("What is your monthly rent?\n"))
utilities = float(input("What is your monthly spending on utilities?\n"))
groceries = float(input("What is your monthly spending on groceries?\n"))
transportation = float(input("What is your monthly spending on transportation?\n"))
savings = float(input("How much money do you plan on saving?\n"))
savings = income/10
total_spending = rent-utilities-groceries-transportation-savings
percent_rent = rent/income*100
percent_utilities = utilities/income*100
percent_groceries = groceries/income*100
percent_transportation = transportation/income*100
percent_spending = total_spending/income*100
percent_savings = savings/income*100
print("You spent $", round(rent,2), "on rent and that is", percent_rent, "% of your income")
print("You spent $", round(utilities,2), "on utilities and that is", percent_utilities, "% of your income")
print("You spent $", round(groceries,2), "on groceries and that is", percent_groceries, "% of your income")
print("You spent $", round(transportation,2), "on transportation and that is", percent_transportation, "% of your income")
print("Every month you should have saved $", round(savings,2), "on savings and that is", percent_savings, "% of your income")

File: Expressions/financial_calc.c
//Tatiana Susov, Financial Calculator C
#include <stdio.h>

int main(void){
 printf("%s\n", "welcome to your personal financial/budget calculater, lets begin!");
 float income, rent, utilities, groceries, transportation, savings, spending,total;
 float pincome, prent, putilities, pgroceries, ptransportation, psavings, pspending;
 printf("What is your monthly income?:");
 scanf("%f", &income);
 printf("What is your monthly rent?:");
 scanf("%f", &rent);
 printf("What is your monthly spending on utilities?:");
 scanf("%f", &utilities);
 printf("What is your monthly spending on transportation?:");
 scanf("%f", &transportation);
 printf("How much do you try to save monthly?:");
 scanf("%f", &savings);
 spending = rent+utilities+groceries+transportation;
 savings = income * .2;
 printf("Your monthly income is $%.2f\n",income);
 total = income-savings-spending;
 printf("Your monthly savings is $%.2f\n",savings);
 printf("Your monthly total is $%.2f\n",total);
 printf("Your monthly spending is $%.2f\n",spending);
 prent=rent/income*100;
 putilities=utilities/income*100;
 pgroceries=groceries/income*100;
 ptransportation=transportation/income*100;
 psavings=savings/income*100;
 pspending=spending/income*100;
 printf("Your expenses are %.1f%% of your income\n",pspending);
 printf("Your rent is %.1f%% of your income\n",prent);
 printf("Your utilities payment is %.1f%% of your income\n",putilities);
 printf("Your grocery spending is %.1f%% of your income\n",pgroceries);
 printf("Your transportation fees is %.1f%% of your income\n",ptransportation);
 printf("Your savings is %.1f%% of your income\n",psavings);
  

    return 0;
}